src/ResourceParser.h

```diff
@@ -113,6 +113,9 @@ namespace snowcrash {
                     ParseResultRef<Attributes> attributes(out.report, out.node.attributes, out.sourceMap.attributes);
                     MarkdownNodeIterator cur = AttributesParser::parse(node, siblings, pd, attributes);
 
+                    // Clear named type context
```

Comment:
Not too sure this comment adds anything? Every word in the comment is in the following function call `namedTypeContext.clear`.